src/frontends/common/include/openvino/frontend/extension/op.hpp

```diff
@@ -145,80 +186,150 @@ class OpConversionFunction {
     std::map<std::string, ov::Any> m_attr_values_map;
 };
 
+class OpConversionFunctionNamed {
```

Comment:
Why do we need this additional class?

Reply:
We need addition information 'input tensor name' and 'output tensor name ' coming from paddle ops to complete the conversion. We will do the specific conversion in 'OpConversionFunctionNamed' which is different from onnx/tensorflow.